src/main/java/core/basesyntax/util/CsvUtil.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.util;
+
+import core.basesyntax.model.DirectoryData;
+
+public class CsvUtil {
```

Comment:
why not some library to write csv file?

Reply:
I think this is not required for this case

Reply:
we will have a separate reference project focused on CSV works